sydent/replication/peer.py

```diff
@@ -115,19 +119,35 @@ def verifyMessage(self, jsonMessage):
             e.requiredServername = self.servername
             raise e
 
+        # Get verify key from signing key
+        signing_key = signedjson.key.decode_signing_key_base64(alg, "0", self.pubkeys[alg])
+        verify_key = signing_key.verify_key
+
+        # Attach metadata
+        verify_key.alg = alg
+        verify_key.version = 0
+
         # Verify the JSON
         signedjson.sign.verify_signed_json(jsonMessage, self.servername, self.verify_key)
 
-    def pushUpdates(self, sgAssocs):
+    def pushUpdates(self, data):
```

Comment:
I think it's a mistake to have the two impls of Peer have different arguments (even if it actually works - I haven't checked)